src/hotspot/share/memory/universe.cpp

```diff
@@ -1248,6 +1248,7 @@ bool Universe::release_fullgc_alot_dummy() {
     if (_fullgc_alot_dummy_next >= fullgc_alot_dummy_array->length()) {
       // No more dummies to release, release entire array instead
       _fullgc_alot_dummy_array.release(Universe::vm_global());
+      _fullgc_alot_dummy_array = OopHandle(); // NULL out OopStorage pointer.
```

Comment:
Seems odd that there is not a way to NULL the pointer directly - why is it not done as part of release()?